### ISSUE TITLE
Reanimate Storybook

### ISSUE THREAD

**Issue by martinfrances107** [jsdrupal/drupal-admin-ui#630]
So this is a quick game of issue queue hopscotch -- where I gather a few issue together and generate a PR that fixes things.

1) An in-range update of storybook is breaking the build 🚨 #582

2) Storybook is broken #546

3) I want to put storybook in a isolated docker container 

1 & 2 -- there is a open PR which partially fixes things but it does not 
pull in all the packages demanded by 

https://storybook.js.org/basics/guide-react/

npm i --save-dev @storybook/react
npm i --save react react-dom
npm i --save-dev @babel/core
npm i --save-dev babel-loader

I have a PR which I am about to make public 

BUT I want to describe a refactoring which is not immediately obvious

From "make up"

there is a lots of parallel processes going on under the hood ... if you start the storybook container  it will automatically do a yarn install and with the node container booting  there will be two container running yarn install on the same volume ... which trouble.

So in short storybook waits for the .yarn-build-complete flag to create by node.

The design decision which at first seems crazy is that I have moved yarn-build-complete into a volume that can be seen by the storybook container. Namely

/var/www/drupal-admin-ui/.yarn-build-complete
